src/Shared/E2ETesting/WaitAssert.cs

```diff
@@ -116,7 +116,7 @@ private static TResult WaitAssertCore<TResult>(IWebDriver driver, Func<TResult>
             // tests running concurrently might use the DefaultTimeout in their current assertion, which is fine.
             TestRunFailed = true;
 
-            var innerHtml = driver.FindElement(By.CssSelector(":first-child")).GetAttribute("innerHTML");
+            var innerHtml = driver.FindElement(By.CssSelector(":first-child"))?.GetAttribute("innerHTML");
```

Comment:
This fails when its not able to start the browser (returns null) so accounting for that